### ISSUE TITLE
Cannot run tests with Jasmine Spec Runner

### ISSUE THREAD

**Issue by petetnt** [adobe/brackets#11773]
I am trying to run tests with Jasmine Spec Runner but opening the Spec Runner just ends up in a blank screen:

![image](https://cloud.githubusercontent.com/assets/7641760/10268449/7273fd22-6ac1-11e5-977b-09ebc84cdd22.png)

Brackets continues to work normally except it cannot be exited anymore without killing the process. Latest master (https://github.com/adobe/brackets/commit/be50533b8575b1f2602f546d01289151e9a671ea), `brackets-shell` shipped that shipped with 1.4. Cannot build the shell myself ATM, so I'll have to wait until `1.5.` to test against that unless someone has some better ideas... Clearing out the cache folder etc. didn't help.

**Comment by abose** [comment]
I am also seeing the same error.

**Comment by abose** [comment]
@le717 https://github.com/adobe/brackets/pull/9622 lead to this break.

Console logs: 
 Failed to load resource: net::ERR_FILE_NOT_FOUND
require.js:141 Uncaught Error: Script error for: project/FileIndexManager
http://requirejs.org/docs/errors.html#scripterror
file:///C:/Users/abose/Documents/brackets/brackets/src/file/NativeFileSystem.js Failed to load resource: net::ERR_FILE_NOT_FOUND
require.js:141 Uncaught Error: Script error for: file/NativeFileSystem
http://requirejs.org/docs/errors.html#scripterror
file:///C:/Users/abose/Documents/brackets/brackets/src/file/NativeFileError.js Failed to load resource: net::ERR_FILE_NOT_FOUND
require.js:141 Uncaught Error: Script error for: file/NativeFileError

**Comment by petetnt** [comment]
Here's the offending lines on `test/SpecRunner.js` 

https://github.com/adobe/brackets/blob/0303dc53b1bc61c7d7305b7f0d8227c4e0804cbf/test/SpecRunner.js#L74

``` js
    // Also load compatibility shims for now, in case legacy extensions are still using old file APIs
    require("project/FileIndexManager");
    require("file/NativeFileSystem");
    require("file/NativeFileError");
    
```

Good call on checking network errors, I just checked the console and didn't see anything :hamburger:

**Comment by petetnt** [comment]
Fixed in #11787, ping @abose
